vagrant_box_defaults.rb

```diff
@@ -8,3 +8,5 @@
 $NETNEXT_SERVER_VERSION= "53"
 @v419_SERVER_BOX= "cilium/ubuntu-4-19"
 @v419_SERVER_VERSION= "12"
+@v49_SERVER_BOX= "cilium/ubuntu"
+@v49_SERVER_VERSION= "170"
```

Comment:
Given that we barely have enough disk space right now, this might blow up in our face (since vagrant will see ubuntu-dev and ubuntu as different and use up another 10GB of space).

Reply:
I think we are running cilium/ubuntu in CI anyway, but let me check.

Reply:
We do.